cmd/node/config/enableEpochs.toml

```diff
@@ -258,7 +258,7 @@
     AutoBalanceDataTriesEnableEpoch = 1
 
     # MigrateDataTrieEnableEpoch represents the epoch when the data tries migration is enabled
-    MigrateDataTrieEnableEpoch = 999999
+    MigrateDataTrieEnableEpoch = 1
```

Comment:
put this on epoch 2? Until we release this RC branch, the flags should be >1

Reply:
Done.